documentation/commands/git-town.md

```diff
@@ -12,6 +12,7 @@ git town hack-push-flag [(true | false)]
 git town help
 git town install-fish-autocompletion
 git town main-branch [<branch_name>]
+git town parent-branch <child_branch_name> <parent_branch_name>
```

Comment:
hmm should this name actually include `feature` since this is just for the parent of feature branches?